README.md

```diff
@@ -5,3 +5,46 @@
 ```shell
 mvn clean package -Pjib-push
 ```
+
+### Packaging the image and push it to local docker daemon for local testing purpose
+
+```shell
+mvn clean package -Pjib-local
+```
+
+### Deploy to [local backend environment](https://github.com/Backbase/local-backend-setup)
+
+Add the following docker compose snippet 
+```yaml
+services:
+    spring-boot-admin-eureka:
```

Comment:
there is no need to set a profile for it to work, the default profile work 100% with eureka, all you need to do is:

```
services:
  spring-boot-admin:
    image: backbasecs/spring-boot-admin-server
    ports:
      - "9080:8080"
    environment:
      <<: *common-variables
```